CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ x.x.x Release notes (yyyy-MM-dd)
 **This project is in the Alpha stage. All API's might change without warning and no guarantees are given about stability. Do not use it in production.**
 
 ### Enhancements
+* Support disableFormatUpgrade ([#310](https://github.com/realm/realm-dart/pull/310))
```

Comment:
[nit] This is clear to me, but is likely not clear for end-users. Might not be super critical right now, but we should aim to keep changelog entries as understandable as possible as for some users they're the first documentation for each new feature. In this case it might be better to have something along the lines of:

```suggestion
* Added a new `Configuration` option: `disableFormatUpgrade`. When set to `true`, opening a Realm with an older file format will throw an exception to avoid automatically migrating it. ([#310](https://github.com/realm/realm-dart/pull/310))
```